test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsTests.cs

```diff
@@ -2042,41 +2042,37 @@ public void LoadSettings_EmptyUserWideConfigFile_DoNotAddNuGetOrg()
                     useTestingGlobalPath: true);
 
                 // Assert
-                var actual = SettingsTestUtils.RemoveWhitespace(File.ReadAllText(Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config")));
-                var expected = SettingsTestUtils.RemoveWhitespace(config);
+                var text = SettingsTestUtils.RemoveWhitespace(File.ReadAllText(Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config")));
+                var result = SettingsTestUtils.RemoveWhitespace(@"<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+    <packageSources>
+        <add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" protocolVersion=""3"" />
+    </packageSources>
+</configuration>");
 
-                actual.Should().Be(expected);
-            }
-        }
+                text.Should().Be(result);
 
-        [Fact]
-        public void LoadSettings_NonExistingUserWideConfigFile_CreateUserWideConfigFileWithNuGetOrg()
-        {
-            using (var mockBaseDirectory = TestDirectory.Create())
-            {
-                // Arrange
-                var nugetConfigPath = Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config");
-                File.Exists(nugetConfigPath).Should().BeFalse();
+                var settingsFile = new SettingsFile(Path.Combine(mockBaseDirectory, "TestingGlobalPath"));
 
                 // Act
-                var settings = Settings.LoadSettings(
-                    root: mockBaseDirectory,
-                    configFileName: null,
-                    machineWideSettings: null,
-                    loadUserWideSettings: true,
-                    useTestingGlobalPath: true);
-
+                var section = settingsFile.GetSection("packageSources");
+                section.Should().NotBeNull();
+                settingsFile.Remove("packageSources", section.Items.First());
+                settingsFile.SaveToDisk();
+
+                settings = Settings.LoadSettings(
+                                    root: mockBaseDirectory,
```

Comment:
nit: maybe reference `settingsFile` to find the path, rather than the directory string directly? Easier to see that this depends on `settingsFile` that way.

Reply:
It's by design that it's not, but this helped me realize that my comment at the top of the PR about the production code being untestable is incorrect.

`settingsFile` is mimicing the user-profile settings directory, but this settings load (with the root directory being different) is mimicing a solution folder.

I'll update the tests to make it all clearer.